lib/es/components/view/selected-items/selected-items.scss

```diff
@@ -63,7 +63,7 @@
     cursor: default;
     font-size: $oc-font-size-normal;
     font-weight: $oc-font-weight-normal;
-    margin: 0.1rem 0 0 2.4rem;
```

Comment:
The same comment as above one

Reply:
I checked that this is not changed. It is defined in the selected-items.scss file and it is not changed.